functional-samples/tutorial.open-api-reference/sw-suggestions.js

```diff
@@ -0,0 +1,24 @@
+import apiList from './api-list.js';
+
+/**
+ * Returns a list of suggestions and a description for the default suggestion
+ */
+export async function getApiSuggestions(input) {
+  const filtered = apiList.filter((api) => api.content.startsWith(input));
```

Comment:
suggestion: rename to better explain what this variable includes
```suggestion
  const suggestions = apiList.filter((api) => api.content.startsWith(input));
```